Website/src/rendering/src/assets/css/base/sitecore.css

```diff
@@ -12,4 +12,12 @@
   .scpm {
     display: none !important;
   }
+
+  main {
```

Comment:
This file is intended for CSS styles in relation to Sitecore tools like Horizon and Experience Editor. Could we move the "main" properties to another file. I have not found a proper file to put it into so we could either create a main.css or layout.css file in the "layout" folder for the main page layout styles.

Reply:
Done.